src/technologies/p.json

```
@@ -1,4 +1,26 @@
 {
+  "Pandectes": {
+    "cats": [
+      67
```

Should we say this is a Shopify app too?

```suggestion
      67,
      100
```